lib/sync/fetcher.go

```diff
@@ -133,6 +135,16 @@ func (f *BlockFetcher) fetch(ctx context.Context, si *SyncInfo) error {
 
 	items, err := f.unmarshalResp(resp.Body)
 	if err != nil {
+		err := errors.Wrap(err, "resp unmarshal")
```

Comment:
Don't use abbreviations in error message. "reponse failed to unmarshal" is much more explanatory.